app/concepts/cinema_halls/use_cases/create_seats.rb

```diff
@@ -0,0 +1,46 @@
+module CinemaHalls
+  module UseCases
+    class CreateSeats
+      attr_reader :params
+
+      def initialize(params:)
+        @params = params
+      end
+
+      def call
+        columns = to_range(@params[:columns])
+        rows = to_range(@params[:rows])
+
+        create_seats(rows, columns)
+      end
+
+      private
+
+      def to_range(array)
+        Range.new(array.first, array.last)
+      end
+
+      def not_available
+        @params[:not_available].map(&:upcase)
+      end
+
+      def seat_available?(seat)
+        not_available.exclude? seat
+      end
+
+      def create_seats(rows, columns)
+        rows.map do |row|
+          columns.map do |column|
+            seat = "#{row}#{column}"
```

Comment:
I think it's a good idea to place kind of separator between row and column. Without that, you will be confused if you have for example 1'st row and 12'th column and an 11'th row and a 2'nd column. Both of that seats will be represented by `112` string. Right?

Reply:
You are right! I was testing creating seats for a hall but until we don't have any specific seat's naming I assume rows are alphabetical and columns are numerical, so I get seats like "A12" and "K2"

Reply:
Ok, I didn't know that you want to use letters for the rows 👍  If you add tests it will be easier to understand the code 😃

Reply:
I haven't thought about that. Thanks!